exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -25,38 +25,41 @@ import (
 	"math"
 	"net/http"
 	"net/url"
+	"strings"
 	"sync"
 
 	"github.com/gogo/protobuf/proto"
 	"github.com/golang/snappy"
 	"github.com/prometheus/prometheus/prompb"
 
+	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/consumer/consumererror"
 	"go.opentelemetry.io/collector/consumer/pdata"
 	"go.opentelemetry.io/collector/internal"
 	otlp "go.opentelemetry.io/collector/internal/data/protogen/metrics/v1"
 	resourcev1 "go.opentelemetry.io/collector/internal/data/protogen/resource/v1"
-	"go.opentelemetry.io/collector/internal/version"
 )
 
 const (
 	maxConcurrentRequests = 5
 	maxBatchByteSize      = 3000000
+	prwVersion            = "0.1.0"
```

Comment:
Why defining this version? Why not using the version from the BuildInfo?

Reply:
Or from internal/version?

Reply:
The version from the BuildInfo would be the version of the Collector. The `prwVersion` is for the X-Prometheus-Remote-Write-Version. Hence, I have defined this version separately.

Reply:
It appears to be the format of the data, something like a content encoding.

But do you need it in the user agent? It's already sent as a header.

Reply:
We need this information as we are using only the User-Agent header information in the back-end AMP service